### PR TITLE
Implement metrics pusher as hosted service

### PR DESCRIPTION
**Pull request by sanych-sun**

### CODE REVIEW

File: src/Prometheus.Client.MetricPusher.HostedService/MetricPusherService.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        async Task DoPushAsync()
+        {
+            try
+            {
+                await _pusher.PushAsync();
+            }
+            catch (Exception e)
+            {
+                // TODO: report error to DiagnosticSource?
```
Comment (sanych-sun) [inline src/Prometheus.Client.MetricPusher.HostedService/MetricPusherService.cs:29]: Need help here. Does anybody have experience with reporting problems into dotnet standard diagnostics system?

File: src/Prometheus.Client.MetricPusher.HostedService/Prometheus.Client.MetricPusher.HostedService.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.*" />
```
Comment (phnx47) [inline src/Prometheus.Client.MetricPusher.HostedService/Prometheus.Client.MetricPusher.HostedService.csproj:8]: Pin dependencies to exact versions. We use Renovate Bot to update them.

**Comment by phnx47** [comment]
@sanych-sun Run `run-me.sh`, it will do some cleanup.

**Comment by phnx47** [comment]
@sanych-sun I did some cleanup before merge. We can discuss later about.

**Review by phnx47 — APPROVED** [review]

**Pull request merged**